src/core/ext/filters/client_channel/health/health_check_client.cc

```diff
@@ -249,11 +255,12 @@ bool DecodeResponse(grpc_slice_buffer* slice_buffer, grpc_error_handle* error) {
 
 HealthCheckClient::CallState::CallState(
     RefCountedPtr<HealthCheckClient> health_check_client,
-    grpc_pollset_set* interested_parties)
+    grpc_pollset_set* interested_parties, MemoryAllocator* allocator)
```

Comment:
I don't think this parameter is necessary, since you can instead access it via `health_check_client_->call_allocator_`.